### PR TITLE
[WIP] Create ReadMe

### PR DESCRIPTION
**Pull request by AdamLalor**
# Description

Please include a summary of the change and which issue is fixed using the headings below

## What

Changed the README file

## Why

To give people an idea and what the website is about and how to contribute to it

## How

How is this change being made

## Verification Steps

What steps were taken to verify the change is working correctly

# Checklist:

- [x] This pr is linked to the related trello card
- [x] My code follows the style guidelines of this project
- [x] I have requested a code review from (at least) two team members
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] This change has been verified by another team member

### CODE REVIEW

**Comment by AdamLalor** [comment]
Hey @ciaranRoche & @dimitraz  would you mind merging this request so that we can have the README on the repo.

**Comment by dimitraz** [comment]
Hey @AdamLalor, do you want to add some more details to the readme? Maybe a title and a description about what this repo is for

**Comment by ciaranRoche** [comment]
@AdamLalor there is already a PR open https://github.com/MountSionCBSSecondary/SchoolWebsite/pull/7 I would focus on getting that PR finished over adding a blank readme to the repo.

**Comment by AdamLalor** [comment]
@ciaranRoche  #there is no readme for me to edit, thats why I'm making this one

**Comment by ciaranRoche** [comment]
@AdamLalor ok let us close out the other PR, and you can use this PR to apply the changes to the Readme, there is no value of adding a blank readme, so just address the comments in the other PR here and we work from there.

**Comment by AdamLalor** [comment]
@ciaranRoche Alright then, I'll start filling in this README

**Comment by ciaranRoche** [comment]
@AdamLalor I have closed your other PR https://github.com/MountSionCBSSecondary/SchoolWebsite/pull/7, so can you ensure you bring over any suggested changes in that PR here, and also update the description of the PR here. Thanks

File: README.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+# Mountsion CBS Secondary Website
+
+This project is a website that is being made by 4 students from Mountsion CBS Secondary along with the help of RedHat Waterford
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. Here's how you can get set up
```
Comment (ciaranRoche) [inline README.md]: This section can be replaced by linking to the tutorial we have in the [docs](https://github.com/MountSionCBSSecondary/SchoolWebsite/blob/master/docs/Tutorial.md)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+Then proceed to copy the link
+
+---
+
+Now open your terminal
+
+Type
+`git clone <url>`
+
+### Terminal
```
Comment (ciaranRoche) [inline README.md]: I feel this section can be removed

File: README.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+`cd..`
+
+The `cd` command is for changing directories, it is used by typing `cd <name-of-file>`
+
+The `ls` command will show all the files in the current directory, this makes it easy to navigate between files
+
+the `cd..` command will go back a directory, if you accidentally go to the wrong file you can use this to go back
+
+### Prerequisites
```
Comment (ciaranRoche) [inline README.md]: As above this section can be removed, feel free to add them to the tutorial doc, and make a separate PR for it

File: README.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+```
+Sublime
+```
+
+```
+Terminator
+```
+
+### Installing
```
Comment (ciaranRoche) [inline README.md]: Again this section can be removed

File: README.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+
+And repeat
+
+```
+until finished
+```
+
+End with an example of getting some data out of the system or using it for a little demo
+
+## Running the tests
```
Comment (ciaranRoche) [inline README.md]: And this one can be removed

File: README.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+
+Explain how to run the automated tests for this system
+
+### Break down into end to end tests
```
Comment (ciaranRoche) [inline README.md]: Can be removed

File: README.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+
+Explain what these tests test and why
+
+```
+Give an example
+```
+
+### And coding style tests
```
Comment (ciaranRoche) [inline README.md]: Can be removed

File: README.md [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+
+Explain what these tests test and why
+
+```
+Give an example
+```
+
+## Deployment
```
Comment (ciaranRoche) [inline README.md]: Can be removed

File: README.md [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+
+Add additional notes about how to deploy this on a live system
+
+## Built With
+
+* [Dropwizard](http://www.dropwizard.io/1.0.2/docs/) - The web framework used
+* [Maven](https://maven.apache.org/) - Dependency Management
+* [ROME](https://rometools.github.io/rome/) - Used to generate RSS Feeds
```
Comment (ciaranRoche) [inline README.md]: Are we using `Dropwizard`, `Maven` and `Rome`?

File: README.md [hunk 10]
```diff
@@ -0,0 +1,128 @@
...
+
+## Contributing
+
+Please read [CONTRIBUTING.md](https://gist.github.com/PurpleBooth/b24679402957c63ec426) for details on our code of conduct, and the process for submitting pull requests to us.
```
Comment (ciaranRoche) [inline README.md]: This link should be to our docs tutorial guide

File: README.md [hunk 11]
```diff
@@ -0,0 +1,128 @@
...
+
+## Versioning
```
Comment (ciaranRoche) [inline README.md]: This can be removed

File: README.md [hunk 12]
```diff
@@ -0,0 +1,128 @@
...
+
+We use [SemVer](http://semver.org/) for versioning. For the versions available, see the [tags on this repository](https://github.com/your/project/tags). 
+
+## Authors
```
Comment (ciaranRoche) [inline README.md]: I like this idea, I think it would be better to change to `Contributors` and we list those who worked on the repo and links to their accounts

File: README.md [hunk 13]
```diff
@@ -0,0 +1,128 @@
...
+
+See also the list of [contributors](https://github.com/your/project/contributors) who participated in this project.
+
+## License
+
+This project is licensed under the MIT License - see the [LICENSE.md](LICENSE.md) file for details
+
+## Acknowledgments
+
+* Hat tip to anyone who's code was used
```
Comment (ciaranRoche) [inline README.md]: I think RedHat mobile should be mentioned here along with your school and the teachers involved

File: README.md [hunk 14]
```diff
@@ -0,0 +1,128 @@
...
+
+## Authors
+
+* **Billie Thompson** - *Initial work* - [PurpleBooth](https://github.com/PurpleBooth)
+
+See also the list of [contributors](https://github.com/your/project/contributors) who participated in this project.
+
+## License
+
+This project is licensed under the MIT License - see the [LICENSE.md](LICENSE.md) file for details
```
Comment (ciaranRoche) [inline README.md]: This link does not work.

File: README.md [hunk 15]
```diff
@@ -0,0 +1,128 @@
+# Mountsion CBS Secondary Website
+
+This project is a website that is being made by 4 students from Mountsion CBS Secondary along with the help of RedHat Waterford
```
Comment (ciaranRoche) [inline README.md]: I think we should reword this as yes it is currently 4 students, but this will be an ongoing project involving students from your school, Also we can acknowledge those who contribute in a section further down the ReadMe

File: README.md (hunk 5 above)
Comment (AdamLalor) [reply README.md]: I haven't edited past this point, I was only using a template

File: README.md (hunk 9 above)
Comment (AdamLalor) [reply README.md]: Like I said above, this was from a template that Laura provided on the old pr

**Comment by ciaranRoche** [comment]
@AdamLalor Just going to edit the title here, I will post this on the slack channel for everyone's attention, just normally add [WIP] to the title to inform people that the PR is still in progress, that way we can give more advice towards the work then giving a review, if that makes sense?

**Comment by AdamLalor** [comment]
@ciaranRoche  Would you mind having a look at this pr when you're ready. For some reason all the text isn't shown in files changed

**Comment by AdamLalor** [comment]
@ciaranRoche would you mind taking a look at this when you get the time to. Thanks

File: README.md [hunk 16]
```diff
@@ -0,0 +1,60 @@
+# Mountsion CBS Secondary Website
+
+![alt text](https://github.com/MountSionCBSSecondary/SchoolWebsite/tree/master/assets/images/Mount Sion Crest.jpg)
```
Comment (ciaranRoche) [inline README.md]: This link is broke, if you rename the image to just `crest.jpg` then you can replace the line to `school logo`
Basically, you linked the full link which won't work you need to link the folder directory path from `README.md` to `crest.jpg`

File: README.md [hunk 17]
```diff
@@ -0,0 +1,60 @@
...
+
+This project is a website that is being made by 4 students from Mountsion CBS Secondary along with the help of RedHat Waterford. As time goes on more students will resume the work that we havn't finished and will continue to keep the website up to date.
+
+## Getting Started
+
+If you wish to contribute to the project take some time to read our getting started guide here
```
Comment (ciaranRoche) [inline README.md]: If you could change this to following `If you wish to contribute to the project take some time to read our [getting started tutorial](./docs/Tutorial.md).` It will provide a direct link to our doc, Sorry I was not clear in my last comment on this.

File: README.md [hunk 18]
```diff
@@ -0,0 +1,60 @@
...
+First you're going to click on "Clone or Download"
+
+Then proceed to copy the link
+
+---
+
+Now open your terminal
+
+Type
+`git clone <url>`
```
Comment (ciaranRoche) [inline README.md]: I think we should remove lines 11 to 28 as we have these steps covered in the docs

File: README.md [hunk 19]
```diff
@@ -0,0 +1,60 @@
+# Mountsion CBS Secondary Website
+
+![alt text](https://github.com/MountSionCBSSecondary/SchoolWebsite/tree/master/assets/images/Mount Sion Crest.jpg)
+
+This project is a website that is being made by 4 students from Mountsion CBS Secondary along with the help of RedHat Waterford. As time goes on more students will resume the work that we havn't finished and will continue to keep the website up to date.
```
Comment (ciaranRoche) [inline README.md]: I think we could maybe reword this again if you don't mind :) `Mount Sion C.B.S school website, built and maintained by transition year students. This project teaches students the fundamentals of web design and development along with project management following the agile methodology. `

File: README.md (hunk 17 above)
Comment (AdamLalor) [reply README.md]: Nah I should have just linked it, my bad.

**Comment by AdamLalor** [comment]
@ciaranRoche Made another push, if you don't mind can you take a look

File: README.md [hunk 20]
```diff
@@ -0,0 +1,41 @@
+# Mountsion CBS Secondary Website
+
+![School Logo](assets/images/crest.jpg)
```
Comment (ciaranRoche) [inline README.md]: You need to rename the image from `Mount Sion Crest.jpg` to `crest.jpg`

**Comment by AdamLalor** [comment]
@dimitraz  @laurafitzgerald  Would one of you mind looking at this before I merge this

File: README.md [hunk 21]
```diff
@@ -0,0 +1,41 @@
...
+
+## Licensed
+  
+This project is licensed under the MIT License - see the [LICENSE.md](https://github.com/MountSionCBSSecondary/SchoolWebsite/blob/master/LICENCE.md) file for details
+
+## Acknowledgments
+
+From all the students involved we would like to thank all those who helped us with special mention to:
+
+* Redhat Waterford
```
Comment (laurafitzgerald) [inline README.md]: Can you add a link to the Red Hat Mobile Twitter here: https://twitter.com/redhatmobile?lang=en

Comment (ciaranRoche) [reply README.md]: +1 on this.
